scripts/README.md

```diff
@@ -0,0 +1,95 @@
+
+4. **ethereum-package**: This script utilizes a modified ethereum-package network configuration, which will be downloaded from [repo](github.com/kurtosis-tech/ethereum-package/) main brunch automatically.
+
+### How to Run
+
+To execute the script with Go, navigate to the directory containing the script and run:
+
+```
+go run emulate_network.go <COMMAND_NAME> [OPTIONS]
+```
+
+Replace `<COMMAND_NAME>` with one of the available commands and provide the necessary options.
+
+### Commands and Options
+To run script `cd` into this (`./scripts`) folder.
+
+1. **build**:
+   - Purpose: Builds a Docker image of the builder.
+   - Options:
+      - `-t`:          (Optional) Image tag for the Docker build. Defaults to `flashbots/builder:dev`.
+      - `-d`           (Optional) Image Build directory. Defaults to `".."`
+      - `-f`           (Optional) Build dockerfile path. Defaults to `"../Dockerfile"`. Use `"./Dockerfile.debug"` for debug capabilities.
+      
+   - Example:
+      ```
+	  go run emulate_network.go build -t=test-builder -f="../Dockerfile" -d=".."
+      ```
+
+
+
+2. **run**:
+   - Purpose: Prepares configurations and starts a Kurtosis enclave.
+   - Options:
+      - `-t`:           (Optional) Image tag. Defaults to `flashbots/builder:dev`.
+      - `-n`:           (Optional) Enclave name. Defaults to `explorer`.
+      - `-a`:           (Optional) Additional builder arguments.
+      - `-s`:           (Optional) Max steps (integer). Defaults to `1000`. Set for `-1` for "unlimited".
+      - `-k`:           (Optional) Path to `kurtosis` executable. Defaults to `kurtosis`.
+      - `--slotTime`:   (Optional) Seconds per slot applied on local devnet. Defaults to 5.
+   - Example:
+     ```
+     go run emulate_network.go run -t=imageTag -a=imageArgs -n=enclaveName -k=/path/to/kurtosis
+     ```
+
+3. **stop**:
+   - Purpose: Stops an active Kurtosis enclave.
+   - Options:
+      - `-k`:           (Optional) Path to `kurtosis` executable. Defaults to `kurtosis`.
+      - `-n`:           (Required) Enclave name.
+   - Example:
+     ```
+     go run emulate_network.go stop -k=/path/to/kurtosis -n=enclaveName
+     ```
+
+4. **help**:
+   - Purpose: Display a summary of available commands and their options.
+   - Example:
+     ```
+     go run emulate_network.go help
+     ```
+## Architecture
+Test network consists of 3 builders and a regular client. Dev and baseline are builders being compared. Dev builder is a candidate to be merged to main while baseline is the current version of builder. The 2 remaining clients are validator which is a builder connected to the relay as validator and a regular el client which submits blocks in epochs 0-3.
+![image](https://github.com/NethermindEth/fb-builder/assets/11379770/54d4a8cf-f64c-4f69-9e32-b35bf43dba0c)
```

Comment:
this image links to the nethermind fb-builder, could add the image as part of the PR?